packages/marketplace/src/core/router.tsx

```diff
@@ -32,9 +32,9 @@ const RegisterConfirm = React.lazy(() => import('../components/pages/register-co
 const AdminStats = React.lazy(() => import('../components/pages/admin-stats'))
 
 const Router = () => {
-  const isReapitEnvProd = process.env.REAPIT_ENV === 'PROD'
+  const isDevelopment = window.reapit.config.appEnv === 'development'
```

Comment:
What about local dev?

Reply:
I think local dev env is the same with development environment so I let 2 things same